src/engine.cpp

```diff
@@ -1110,6 +1112,11 @@ void EnginesMonthlyLoop()
 	}
 }
 
+static IntervalTimer<TimerGameCalendar> _engines_monthly({TimerGameCalendar::MONTH, TimerGameCalendar::Priority::ENGINE}, [](auto)
```

Comment:
Shouldn't this line practically replace the `EnginesMonthlyLoop()` like it does for `EnginesDailyLoop()`?

Reply:
Yes, except for some reason this function is also called from the cheats menu.